File: app/src/main/java/com/neome/java/api/meta/base/dto/StudioDtoChartYAxis.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Symbol;
import com.neome.java.api.meta.base.Types.MetaIdChartYAxis;
import com.neome.java.api.meta.base.Types.MetaIdField;
import com.neome.java.api.meta.base.Types.MetaIdVar;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioDtoChartYAxis extends StudioBase
{
  @Nullable
  public StudioDtoColor color;

  @Nullable
  public MetaIdField colorFieldId;

  @Nullable
  public MetaIdVar colorVarId;

  @Nullable
  public MetaIdField fieldId;

  @Nullable
  public String label;

  public MetaIdChartYAxis metaId;

  @Nullable
  public Symbol name;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioDtoChartYAxis.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdChartYAxis
import com.neome.api.meta.base.Types.MetaIdField
import com.neome.api.meta.base.Types.MetaIdVar
import com.neome.api.meta.base.dto.StudioBase
import com.neome.api.meta.base.dto.StudioDtoColor
import com.neome.api.meta.base.Symbol

interface StudioDtoChartYAxis : StudioBase
{
  val color: StudioDtoColor?
  val colorFieldId: MetaIdField?
  val colorVarId: MetaIdVar?
  val fieldId: MetaIdField?
  val label: String?
  val metaId: MetaIdChartYAxis
  val name: Symbol?
}